src/apps/views/AppPermissionRequestView/usePermissionsRequestRedirects.ts

```diff
@@ -0,0 +1,36 @@
+import { AppPaths } from "@dashboard/apps/urls";
+import useNavigator from "@dashboard/hooks/useNavigator";
+
+const errors = {
+  USER_DENIED_PERMISSIONS: "USER_DENIED_PERMISSIONS",
+};
+
+// todo: test
```

Comment:
**nitpick** - looks like the test is already written, so this comment can be removed